docs/servo_power_monitor/src/index.ts

```diff
@@ -638,5 +638,6 @@ function setupDataLoad() {
   dropZone.innerText = 'Drop .json here';
   dropZone.addEventListener('dragover', handleDragOver, false);
   dropZone.addEventListener('drop', handleFileSelect, false);
+  console.log("hello!!!!!!!");
```

Comment:
Is this a debug message?

Reply:
aaaahhhh, good catch... let me remove that...

Reply:
done! Please review again ;)